### ISSUE TITLE
Add info on where to access data in Primer 3

### ISSUE THREAD

**Issue by stephwright** [mozillascience/open-data-training#27]
John Kratz commented: I think it would make more sense to merge geospatial and temporal coverage into one item and break off access/licensing/reuse.

Amel Ghouila: I perfectly agree

Stephanie Wright:I'm wondering what that will do to the flow of the original six questions we asked (WHY, WHAT, WHO, WHERE, WHEN & HOW). We can combine WHEN & WHERE into one section w/o too much trouble. If we break out access/licensing/reuse, we are breaking the questions out across two sections and w/i in one section at the same time. (Geospatial in one section and location of access in another while both address WHERE; combining access/licensing/reuse into one section gives us a section containing WHERE & HOW(?))

John Kratz:  Okay, I can see the logic of the current arrangement, but I still think combining the two senses of WHERE is confusing. If we were talking about physical specimens collected here and stored there, that would make sense, but the WHERE of the dataset will almost always be a URL (or something that resolves to a URL), and of course we don't care about the physical location.

What if we pulled access/license stuff from this primer and put it in primer 3? The only reason we're talking about it here is that it should go in the DATA_README, but you have to go through the next primer to have that information anyway. Maybe just put a note at the end of the six questions that there's one more thing to add, but we're not ready for that yet and we'll get to it in the next primer?

TLDR version: I think we should divide the discussion of metadata into descriptive metadata and administrative metadata, and cover descriptive here and administrative in primer 3.
